libprecompiled/Common.h

```diff
@@ -41,6 +41,11 @@ namespace precompiled
 
 enum PrecompiledError : int
 {
+    // PaillierPrecompiled -51699 ~ --51600
+    CODE_DIFFERENT_PUBKEYS = -51602,
+    CODE_INVALID_CIPHERS = -51601,
+    CODE_UNDEFINED_FUNC = -51600,
```

Comment:
what about PR with the implement of PaillierPrecompiled?

Reply:
because of the limitation of 250 lines~